src/bp/ui-admin/src/Pages/Bot/Details.jsx

```diff
@@ -19,7 +19,7 @@ import api from '../../api'
 import PageContainer from '~/App/PageContainer'
 import StickyActionBar from '~/App/StickyActionBar'
 import { Button, Intent } from '@blueprintjs/core'
-import confirmDialog from '~/App/ConfirmDialog'
+import { confirmDialog } from '../../../../react-botpress-components/dist'
```

Comment:
Can you add an alias for this? Ex: botpress/ui-shared

Reply:
Oops forgot to change it